packages/react-form-state/src/validators.ts

```diff
@@ -97,12 +96,12 @@ export function validateList<Input extends Object, Fields>(
 export function validate<Input>(
   matcher: Matcher<Input, any>,
   errorContent: ErrorContent,
-): (input: Input) => ValidationFunction<Input, never>;
```

Comment:
this type was just wrong (probably wasn't breaking anything because validationFunction can return `any` anyway)